sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -264,8 +281,7 @@
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<ConfigurationSetting> getSetting(String key, String label, OffsetDateTime asOfDateTime) {
         return withContext(context -> getSetting(new ConfigurationSetting().setKey(key).setLabel(label), asOfDateTime,
-                false, context))
-            .flatMap(response -> Mono.justOrEmpty(response.getValue()));
+                false, context)).map(response -> response.getValue());
```

Comment:
There is a FluxUtil method to handle pulling a response value out, I believe it is called toMono. Should use that in all the places update.